src/types/index.ts

```diff
@@ -27,6 +27,7 @@ export enum ArbiterAccount {
   TWITTER = "TWITTER",
   ENS = "ENS",
   EMAIL = "EMAIL",
+  FARCASTER = "FARCASTER",
 }
 
 export enum ChallengerAccount {
```

Comment:
Why is this different from `ArbiterAccount` and why does it have different values? 🤔 
We should support the same for both, ideally let's coalesce these into one enum, say `AccountType` or `OnboardingType`

Reply:
You mean `ChallengerAccount`? The only difference is the cost actions the accounts will need to take. Arbiters only perform 'free' (sponsored) actions, but challengers also need to put in the wager, which might require having / buying crypto. With this logic, we also shouldn't allow twitter. Let's discuss offline